core/src/main/java/fund/cyber/markets/common/CircularQueue.java

```diff
@@ -28,4 +28,8 @@ public void addNext(T element) {
     public T[] getElements() {
         return elements;
     }
+
+    public T getElement(Integer index) {
```

Comment:
Prefer `int` over `Integer`, since `int` is `Int` in kotlin, and `Integer` is `Int!`.

Reply:
done